File: python/src/tests/t_cleanup_locks.py
#!/usr/bin/python3
#
# $Id:$
#
# We test a bit of the atrshmlog here.
#
# This is for the first starter, so only the basic things.

import sys
import atrshmlog

r = atrshmlog.attach()

print('cleanup locks : ' + str(r) + ' : ')

area = atrshmlog.get_area()

print('cleanup locks : ' + str(area) + ' : ')

atrshmlog.cleanup_locks(area)

print('cleanup locks ')



print (' ')

exit(0);

# end of test

File: perl/src/tests/t_cleanup_locks.pl
#!/usr/bin/perl
#
# $Id:$
#
# perl test driver

use Atrshmlog;

$result = Atrshmlog::attach();

print "cleanup locks : $result : \n";

$area = Atrshmlog::get_area();

print "cleanup locks : $area : \n";

Atrshmlog::cleanup_locks($area);

print "cleanup locks \n";

print " \n";

exit (0);

# end of main
